CHANGELOG.md

```diff
@@ -6,6 +6,24 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 For releases `< 1.0.0` minor version steps may indicate breaking changes too.
 
+## [Unreleased] - YYYY-MM-DD
+
+### Added
+
+### Changed
+
+### Removed
+
+### Fixed
+
+- Fixed inconsistent base image and build image versions
+
+### Security
+
+### Deprecated
+
+### Known issues
+
```

Comment:
Please update the changelog. Reference the PRs/issues in the upstream repos.

Reply:
It is updated now.
If you have any comments, please let me know.